packages/launchdarkly-adapter/modules/adapter/adapter.ts

```diff
@@ -31,7 +32,7 @@ type LaunchDarklyAdapterState = {
 const adapterState: TAdapterStatus & LaunchDarklyAdapterState = {
   isReady: false,
   isConfigured: false,
-  isUnsubscribed: false,
+  subscriptionStatus: TAdapterSubscriptionStatus.Subscribed,
```

Comment:
Screw booleans. Another nite the others will become enums too. Sadly they can't be const enums due to babel which affects bundle size a bit.